.env.example

```diff
@@ -2,4 +2,8 @@
 # If no variable found, DICOMWeb panel in data tab is hidden unless set in settings.
 VUE_APP_DICOM_WEB_URL=http://localhost:5173/dicom-web
 # Display name of DICOMWeb host
-VUE_APP_DICOM_WEB_NAME=ACME Hospital
\ No newline at end of file
+VUE_APP_DICOM_WEB_NAME=ACME Hospital
+
+VUE_APP_SENTRY_DSN=xhttps://330d4ca81b9c48cfac837d395966e6ee@o4505347877765120.ingest.sentry.io/4505347880779776
+
+VUE_APP_ENABLE_REMOTE_SAVE=true
```

Comment:
I like this idea generally. There are two thoughts here:
- builds must explicitly set this to true
- the save dialog puts up a warning if there is a saveUrl

By the way, I don't see this env var used anywhere in this PR.

Reply:
Got it. I must have missed it before.

Let's also add a few comments describing what each env var does.